### PR TITLE
feat(collector-options-form): add No Options case

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

image

image

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue [hunk 1]
```diff
@@ -59,7 +82,9 @@ const props = defineProps<{
 const emit = defineEmits<{(e: 'update:isValid', isValid: boolean): void;}>();
 
 const state = reactive({
+    isVisibleTitle: computed<boolean>(() => (!props.hasMetadata && !Object.keys(state.schema ?? {}).length)),
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue:85]: suggestion: What about to add `showTitleIfNoSchema` prop, and replace `!props.hasMetadata` with `props.showTitleIfNoSchema`?

**Review by WANZARGEN — APPROVED** [review]
Looks good!

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep1.vue [hunk 2]
```diff
@@ -142,7 +142,7 @@ watch([() => collectorFormState.provider, () => state.selectedRepository], async
 }, { immediate: true });
 
 onMounted(() => {
-    collectorFormStore.resetForm();
+    collectorFormStore.$reset();
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep1.vue:145]: praise: Thank u !!

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep3.vue [hunk 3]
```diff
@@ -3,6 +3,7 @@
         <div class="input-form">
             <attached-service-account-form @update:isAttachedServiceAccountValid="handleChangeIsAttachedServiceAccountValid" />
             <collector-options-form class="collector-options-form"
+                                    show-title-if-no-schema
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep3.vue:6]: praise: I love it!

File: apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue [hunk 4]
```diff
@@ -76,13 +76,14 @@ const collectorFormStore = useCollectorFormStore();
...
-    hasMetadata?: boolean;
+    hasMetadata?: boolean; // MEMO: if true, use metadata(state.schema) of collectorFormState.originCollector. And if false, call api for get metadata(state.schema).
+    showTitle?: boolean;
     resetOnCollectorIdChange?: boolean;
 }>();
 const emit = defineEmits<{(e: 'update:isValid', isValid: boolean): void;}>();
 
 const state = reactive({
-    isVisibleTitle: computed<boolean>(() => (!props.hasMetadata && !Object.keys(state.schema ?? {}).length)),
+    isVisibleTitle: computed<boolean>(() => (!!props.showTitle && (!props.hasMetadata && !Object.keys(state.schema ?? {}).length))),
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue:86]: suggestion:
```suggestion
    isVisibleTitle: computed<boolean>(() => (!!props.showTitle && !Object.keys(state.schema ?? {}).length))),
```

File: apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue [hunk 5]
```diff
@@ -76,13 +76,14 @@ const collectorFormStore = useCollectorFormStore();
 const collectorFormState = collectorFormStore.$state;
 
 const props = defineProps<{
-    hasMetadata?: boolean;
+    hasMetadata?: boolean; // MEMO: if true, use metadata(state.schema) of collectorFormState.originCollector. And if false, call api for get metadata(state.schema).
+    showTitle?: boolean;
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue:80]: nitpick: more explicit name would be nice like 'showTitleOnEmptySchema' :)

**Review by WANZARGEN — APPROVED** [review]
Thank you for apply requirements of detail page. :)

**Pull request merged**